cli.js

```diff
@@ -24,41 +26,49 @@ const cli = meow(`
 	Run without arguments to use the interactive interface.
 `, {
 	alias: {
-		f: 'force'
+		f: 'force',
+		v: 'verbose'
 	}
 });
 
-function init() {
+function init(flags) {
 	escExit();
 
-	return psList({all: false}).then(listProcesses);
+	return psList({all: false}).then(procs => listProcesses(procs, flags));
 }
 
-function listProcesses(processes) {
+function listProcesses(processes, flags) {
 	inquirer.registerPrompt('autocomplete', require('inquirer-autocomplete-prompt'));
 
 	return inquirer.prompt([{
 		name: 'processes',
 		message: 'Running processes:',
 		type: 'autocomplete',
-		source: (answers, input) => Promise.resolve().then(() => filterProcesses(input, processes))
+		source: (answers, input) => Promise.resolve().then(() => filterProcesses(input, processes, flags))
 	}])
 		.then(answer => fkill(answer.processes))
 		.then(init);
 }
 
-function filterProcesses(input, processes) {
+function filterProcesses(input, processes, flags) {
+	const filters = {
+		name: proc => input ? proc.name.toLowerCase().includes(input.toLowerCase()) : true,
+		verbose: proc => input ? proc.cmd.toLowerCase().includes(input.toLowerCase()) : true
+	};
 	return processes
-		.filter(proc => input ? proc.name.toLowerCase().includes(input.toLowerCase()) : true)
+		.filter(flags.verbose ? filters.verbose : filters.name)
 		.sort((a, b) => numSort.asc(a.pid, b.pid))
-		.map(proc => ({
-			name: `${proc.name} ${chalk.dim(proc.pid)}`,
-			value: proc.pid
-		}));
+		.map(proc => {
+			const lineLength = process.stdout.columns || 80;
+			const margins = 4 + proc.pid.toString().length;
```

Comment:
What is `4`? Can you make it a constant with a descriptive name? It seem arbitrary without more context. Also note that `cliTruncate` accounts for the dots, so you don't have to do that.